src/assets/StoryEvents.ts

```diff
@@ -35,6 +35,10 @@ export const StoryEvents: Event[] = [
                         fromPublic: `Finally on the right track <i class='feedback-message-emoji fas fa-subway'></i>`,
                         fromBusiness: `CONFIDENCE IN ACHIEVING A STABLE ECONOMY RISES`,
                         fromHealthcare: `COVID-19 cases rising sharply!`
+                    },
```

Comment:
The story events now have the possibility of defining containment policies and improvements that will be used as inputs for the simulator.